linux_os/guide/services/ntp/chronyd_client_only/rule.yml

```diff
@@ -0,0 +1,30 @@
+documentation_complete: true
+
+prodtype: rhel8,fedora,ol8
+
+title: 'Disable chrony daemon from acting as server'
+
+description: |-
+    The <tt>port</tt> option in <tt>/etc/chrony.conf</tt> can be set to
+    <tt>0</tt> to make chrony daemon to never open any listening port
+    for server operation and to operate strictly in a client-only mode.
+
+rationale: |-
+    Minimizing the exposure of the server functionality of the chrony
+    daemon diminishes the attack space.
```

Comment:
nitpick: I think a more broadly used term is `attack surface`.

Reply:
Fixed in 430c2d727d117cacff5ff0ebd50df78cc3671251.